frontend/src/modules/entitydetails/components/Metadata.js

```diff
@@ -310,7 +310,7 @@ export default class Metadata extends React.Component<Props, State> {
                 id='entitydetails-Metadata--placeholder'
                 attrs={{ title: true }}
             >
-                <Property title='Placeholder Examples' className='placeholder'>
+                <Property title='PLACEHOLDER EXAMPLES' className='placeholder'>
```

Comment:
The failing test is looking for this element based in part on this title. So, you will need to update the test to reflect the uppercase status of this title in order for that test to pass.